definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json

```diff
@@ -93,35 +93,35 @@
   {
     "LiveFrom": "24/11/2022",
     "CaseTypeID": "PRLAPPS",
-    "CaseFieldID": "ordersSubmittedWithApplication",
+    "CaseFieldID": "draftOrders",
```

Comment:
Please keep the "ordersSubmittedWithApplication" , casefield, Reason being any case who has got this casefield in the casedata will throw the CaseValidationException , if it is removed away .